config/runtime.exs

```diff
@@ -28,23 +28,7 @@ if config_env() == :prod do
   {:ok, _} = Application.ensure_all_started(:ex_aws)
   {:ok, _} = Application.ensure_all_started(:ex_aws_secretsmanager)
 
-  secret_key_base =
-    (eb_env_name <> "-secret-key-base")
-    |> ExAws.SecretsManager.get_secret_value()
-    |> ExAws.request!()
-    |> Map.fetch!("SecretString")
-
-  api_v3_key =
-    (eb_env_name <> "-api-v3-key")
-    |> ExAws.SecretsManager.get_secret_value()
-    |> ExAws.request!()
-    |> Map.fetch!("SecretString")
-
-  screens_auth_secret =
-    (eb_env_name <> "-screens-auth-secret")
-    |> ExAws.SecretsManager.get_secret_value()
-    |> ExAws.request!()
-    |> Map.fetch!("SecretString")
+  eb_env_name = System.get_env("ENVIRONMENT_NAME")
```

Comment:
This is already set on line 7. No need to do it again, right?

Reply:
Ahh yes thanks. Didn't see that up there.